### DISCUSSION TITLE
Multiple errors when running Paparazzi tests

### DISCUSSION THREAD

**Discussion by akaczorowski** [cashapp/paparazzi#337] [Q&A]
Hi,

reaching for help here, not sure why but I am receiving multiple errors in Paparazzi tests, can you help me?
Sample project: [MyApplicationv3.zip](https://github.com/cashapp/paparazzi/files/7607231/MyApplicationv3.zip)
 Errors:
 

> com.example.feedback.FirstViewTest > simple FAILED
>     java.lang.ExceptionInInitializerError at Build.java:229
>         Caused by: java.lang.ArrayIndexOutOfBoundsException at Build.java:214
> 
> com.example.feedback.FirstViewTest > simple2 FAILED
>     java.lang.NullPointerException at Layout.java:453
> 
> com.example.feedback.FirstViewTest > simple3 FAILED
>     java.lang.NullPointerException at Layout.java:453
> 
> com.example.feedback.PaparazziExampleTest > simpleTestPaparazzi FAILED
>     java.lang.NullPointerException at Layout.java:453

 Secondly, I am also trying to setup Paparazzi in another project and receiving even more weird errors e.g.
> java.lang.ClassCastException: class com.android.layoutlib.bridge.MockView cannot be cast to class android.widget.TextView (com.android.layoutlib.bridge.MockView and android.widget.TextView are in unnamed module of loader 'app')

 at method call:

> configureTitlev2(viewGroup.findViewById(R.id.title), configuration)

> private fun configureTitlev2(
>     textView: TextView,
>     configuration: ToolbarConfiguration
> ) { ...}

**Comment by edenman** [comment]
Did you ever figure out the `class com.android.layoutlib.bridge.MockView cannot be cast to` issue?
